Products/ZenHub/server/executors/tests/test_workers.py

```diff
@@ -79,12 +79,17 @@ def test_create(t, _zhwlist, _ps, _mp):
         t.assertIs(result._WorkerPoolExecutor__worklist, _zhwlist.return_value)
         t.assertIs(result._WorkerPoolExecutor__workers, pool)
 
+    def test_create_requires_pool(t):
+        with t.assertRaises(ValueError):
+            WorkerPoolExecutor.create(t.name)
```

Comment:
I wrote a test, `test_create_fails_on_missing_args`, which covers all the missing args to `create` scenarios.  Using the "custom" `subTest` context manager.

Perhaps this test is redundant.

Reply:
removed redundant test